dashboard/db/migrate/20181117002905_rename_penultimate_dance_level.rb

```diff
@@ -0,0 +1,14 @@
+class RenamePenultimateDanceLevel < ActiveRecord::Migration[5.0]
+  def up
+    old_name = 'Dance_Party_11.5'
+    new_name = 'Dance_Party_11_5'
+    raise "level '#{new_name}' already exists" if Level.find_by(name: new_name)
```

Comment:
do we want to raise, or just skip this silently? I can see this being problematic for running local dashboard migrations in the future